### PR TITLE
Disabling network access

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 1]
```diff
@@ -7,24 +7,53 @@
...
+AuraDB Virtual Dedicated Cloud and AuraDS Enterprise run in a dedicated AWS cloud account, Azure subscription or GCP project to achieve complete isolation for your deployment.
+Additional Virtual Private Cloud (VPC) boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected. 
+The Aura console resides in a separate VPC, isolated from the rest of the Aura services.
 
-Additional VPC boundaries enable you to operate within an isolated section of the service, where your processing, networking, and storage are further protected.
+== Network access
 
-The Aura Console runs in a separate VPC, separate from the rest of Aura.
+An Aura instance can be publicly available, completely private, or configured to allow both public and private access.
+To configure network access, you need to be authorized to access the part of [your system's?] infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC. This includes the ability to connect over the cloud provider's private link and private endpoint.
```
Comment (fiquick) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:17]: @AlexicaWright do you think the original writer meant authorized access to the "customer insfrastructure", "Aura infrastructure" or both?

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 2]
```diff
@@ -7,24 +7,52 @@
...
-If your Aura instances are public, traffic to them is allowed to traverse the public internet and they are accessible with the correct username and password.
+The Aura console provides a step-by-step configuration guide to:
 
-For your instance to be completely private, turn public traffic off, use the cloud provider's network, and create a private endpoint inside your VPC, which gives you a private connection to Aura.
-The only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create.
+. Choose your Aura instance details
+. Create a VPC endpoint
+. Accept endpoint connection requests and enable private DNS in the cloud provider's console
+. Disable public traffic (optional). 
+If you disable public traffic it is highly recommended to link:https://support.neo4j.com/s/article/13174783967507-How-To-Test-Connectivity-Through-The-Private-Endpoint[Test Connectivity Through The Private Endpoint]
```
Comment (fiquick) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:30]: I was wondering, if I should put the steps in here? They are available in a KB and it's a major pain point that customer's are not checking their connectivity.

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 3]
```diff
@@ -7,24 +7,52 @@
...
+==== Disabling public traffic
+
+You can return to Step 4 at any time to disable public traffic, even if you’ve already completed the network access configuration and initially allowed public traffic. 
+To do this, click through the steps in the network access configuration guide until you reach Step 4, where you’ll find the option to disable public traffic.
+
+Note: after disabling public traffic - the disabling does not happen immediately, you will see the status change in the console when it is complete [see Fig.1].
+
+.Network access visual
+[.shadow]
+image::network_access.png[]
```
Comment (fiquick) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:41]: The shield icon is a bit wonky in UPX designs.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 4]
```diff
@@ -65,7 +93,7 @@ Aura VPCs can't initiate connections back to your VPCs.
 ==== Browser and Bloom access over private endpoints
 
 To connect to your instance via Browser or Bloom, you must use a dedicated VPN.
-This is because when you disable public access to your instance, this applies to all connections, including those from your computer when using Browser or Bloom.
+This is because when you xref:[disable public access] to your instance, this applies to all connections, including those from your computer when using Browser or Bloom.
```
Comment (fiquick) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:96]: Do you think I should link to the disabling public access section here?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc (hunk 4 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/platform/security/secure-connections.adoc:97]: Yes, why not? I think it would be helpful.

**Review by AlexicaWright — COMMENTED** [review]
I don't know about the image though.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Comment by fiquick** [comment]
Hi Jessica, what do you reckon, do you think this in a fit state to merge?

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 5]
```diff
@@ -7,24 +7,40 @@
 label:AuraDB-Virtual-Dedicated-Cloud[]
 label:AuraDS-Enterprise[]
 
-AuraDB Virtual Dedicated Cloud and AuraDS Enterprise run in a dedicated cloud Account (AWS), Subscription (Azure) or Project (GCP) to achieve complete isolation for your deployment.
+AuraDB Virtual Dedicated Cloud and AuraDS Enterprise run in a dedicated AWS cloud account, Azure subscription or GCP project to achieve complete isolation for your deployment.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:10]:
```suggestion
AuraDB Virtual Dedicated Cloud and AuraDS Enterprise run in a dedicated AWS cloud account, Azure subscription, or GCP project to achieve complete isolation for your deployment.
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 6]
```diff
@@ -7,24 +7,40 @@
...
 
-The Aura Console runs in a separate VPC, separate from the rest of Aura.
+An Aura instance can be publicly available, completely private, or allow both public and private access.
 
-== Network access
+If private traffic is enabled, your Aura instances are public and traffic to them is allowed to traverse the public internet and they are accessible with the correct username and password.
 
-An Aura instance can be publicly available, completely private, or both.
-To configure this, you need to be authorized to access the part of the infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC.
+To make your instance completely private, disable public traffic, use the cloud provider's network and create a private endpoint inside your VPC, which gives you a private connection to Aura.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:20]:
```suggestion
To make your instance completely private, you need to disable public traffic, use the cloud provider's network, and create a private endpoint inside your VPC, which gives you a private connection to Aura.
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 7]
```diff
@@ -7,24 +7,40 @@
...
+The only way to connect to your database is from inside your network (your VPC in your AWS/Azure/GCP account) using an internal IP address you choose and DNS records you create.
+
+To configure network access, you need to be authorized to access the part of the infrastructure that runs and handles these instances as well as the networking used to establish secure connections between the database and the application's VPC. 
 This includes the ability to connect over the cloud provider's private link and private endpoint.
 
-If your Aura instances are public, traffic to them is allowed to traverse the public internet and they are accessible with the correct username and password.
+To configure settings for network access to your instance go to *Aura console* > *Security* > *Network access* > *New network access configuration*
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:26]:
```suggestion
To configure settings for network access to your instance, go to *Aura console* > *Security* > *Network access* > *New network access configuration*.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

File: modules/ROOT/pages/platform/security/secure-connections.adoc (hunk 4 above)
Comment (AlexicaWright) [reply modules/ROOT/pages/platform/security/secure-connections.adoc]: You'll need to insert the xref here.

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 8]
```diff
@@ -260,6 +276,40 @@ The private ingress then directs the connection to the Aura instance with dbid:
 |role: read
 |=== 
 
+== Test connectivity through the private endpoint
+
+Utilize the `nslookup` command to confirm whether the Fully Qualified Domain Names (FQDNs) of your Aura instances are directed to the IP address of the PrivateLink endpoint (usually represented by an internal IP address, such as 10.0.0.0).
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:281]:
```suggestion
Use the `nslookup` command to confirm whether the Fully Qualified Domain Names (FQDNs) of your Aura instances are directed to the IP address of the PrivateLink endpoint (usually represented by an internal IP address, such as 10.0.0.0).
```

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 9]
```diff
@@ -260,6 +276,40 @@ The private ingress then directs the connection to the Aura instance with dbid:
...
+
+[source,bash]
+----
+nslookup <dbid>.production-orch-<orchestra>.neo4j.io
+----
+
+. Use curl from a VM instance or a container located in the related VPC network.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:288]:
```suggestion
. Use cURL from a VM instance or a container located in the related VPC network. 
```

**Review by AlexicaWright — COMMENTED** [review]
A few suggestions.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Review by AlexicaWright — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-536.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-536.surge.sh/aura/classic/platform/security/secure-connections)

**Pull request merged**